modules/ROOT/pages/swift.adoc

```diff
@@ -1088,6 +1088,10 @@ The Couchbase Lite API is thread safe except for calls to mutable objects: `Muta
 +
 xref:index.adoc[Read more]
 
+*macOS Support*
+
+Support for macOS 10.9 and 10.10 is being deprecated in this release and will be removed in a future release (see <<supported-versions>>).
```

Comment:
```suggestion
Support for macOS 10.9 and 10.10 is being deprecated in this release and will be will be removed after two (non-maintenance) releases following the deprecation announcement.  (see <<supported-versions>>).
```

Reply:
@jamiltz Also, thinking a little more, I think we should replace "in this release"in the suggested line with "version 2.6" . That way this statement would also apply to 2.7 and beyond.